src/signalr/azext_signalr/_client_factory.py

```diff
@@ -0,0 +1,14 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def _signalr_client_factory(cli_ctx, *_):
```

Comment:
It seems unnecessary to have this function as the only reference is in `cf_signalr`. These two methods can be merged.

Reply:
It is the general pattern though used in other modules. Makes it easy to add new client factories later.